src/codegen/riscv64/assembler-riscv64.cc

```diff
@@ -1815,6 +1815,35 @@ void Assembler::c_addi(Register rd, int8_t imm6) {
   GenInstrCI(0b000, C1, rd, imm6);
 }
 
+void Assembler::c_addiw(Register rd, int8_t imm6) {
```

Comment:
Can you use `zero_reg` for `ToRegister(0)`, `sp` for `ToRegister(2)` (in other places too)? It may make the codes easier to understand.